R/methods.R

```diff
@@ -40,6 +29,21 @@ format.lint <- function(x, ..., width = getOption("lintr.format_width")) {
   )
 }
 
+build_line_ref <- function(x) {
+  line_ref <- paste0(
+    x$filename, ":",
+    as.character(x$line_number), ":",
+    as.character(x$column_number)
+  )
+
+  create_clickable_link <- cli::ansi_has_hyperlink_support()
```

Comment:
Suggestion: this might read nicer as:

```r
if (!cli::ansi_has_hyperlink_support()) {
  return(line_ref)
}
cli::format_inline("{.path {line_ref}}")
```

WDYT?
```